### DISCUSSION TITLE
GitHub Classroom: assignment delete button unresponsive

### DISCUSSION THREAD

**Discussion by GMU-instructor** [community/community#64548] [GitHub Education]
### Hi everyone,

Hello. I recently set up a new Github Classroom and added some assignments to test it. Now, I'm trying to delete the assignments and the delete button is unresponsive. I've seen several prior similar issues raised with this function, but those discussions have been archived. I go to the Github Classroom, click on Assignments, click the delete (trash can) icon, and modal box pops up. It has the 'unexpected things will happen if you don't read this' statement, then the list of disclaimers about what will happen when I delete. But, there is no input box for me to type anything to confirm. I am assuming the button is not working because I am not typing the name of the assignment to confirm, but there is no box for me to type it. Pasted screenshot of the modal below. I cannot scroll the modal. I have tried to resize the whole browser but that doesn't fix it either. I have tried in both Chrome and Edge. And I tried restarting my laptop.

![image](https://github.com/community/community/assets/94993907/e02a50d9-825c-47cc-8417-f2269aaeae8a)

**Comment by arafajs** [comment]
- Try deleting the assignment from a different browser.
- Try clearing your browser's cache and cookies.
- Try disabling any extensions that you are using in your browser.

**Reply by tcpan** [reply]
I experienced the same issue.  tried with firefox, MS edge, and iOS safari.   tried with private browsing too.  safari test was on a device that never visited that site and does not have any extensions.

**Reply by dendy-prtha-tiukdw** [reply]
hi, face same issue when try to delete assignment. if there is any solution for it...?

**Reply by susaneileenfox** [reply]
This is also happening to me. Both in Chrome and in Firefox, Clicking the "I have read and understand these effects" does nothing

**Reply by debbieyuster** [reply]
Same issue. And also a different issue that led me to want to delete the assignment in the first place. Reusing an assignment from another Github Classroom is leading to the assignment being created in the new classoom, but without the template code copied over - instead I get the error message:

> The assignment copy process has failed: We were unable to copy your assignment Homework 1. Please try again or [contact support](https://support.github.com/contact/education)

**Reply by Pieter-hogent** [reply]
Same here, different browsers, different assignments, button appears to not work

**Reply by dburchill** [reply]
any fix for this? I'm having the same problem
